docs/gradle-plugin/configuring.md

```diff
@@ -0,0 +1,432 @@
+By specifying an empty filter `filters { }`, you can completely disable report filtering.
+
+## Class name with wildcards
+
+Inclusion/exclusion value rules:
+
+* Can be a fully-qualified class name.
+* Can contain wildcards:
+    * `*` for zero or several of any char.
+    * `**` is the same as `*`.
+    * `?` for one of any char.
+* File and directory names are not allowed.
+
+Examples:
+
+* (good) `my.package.ClassName`
+* (good) `my.*.*Name`
+* (bad) `my/package/ClassName.kt`
+* (bad) `src/my.**.ClassName`
+
+## Verification
+When checking a certain verification rule, the entire code is divided into units of code for which it determines whether it was covered (executed) or skipped (not executed).
+For example, an entire line from source code or a specific JVM instruction from compiled byte-code can be executed or not.
+
+All units are grouped into one or more groups. 
+Based on amount of the executed and non-executed code units, one number (current value) will be calculated for each group using the aggregation function.
+
+It is this current value of the group that will be compared with the minimum and maximum boundaries specified in the configuration.
+
+Type `MetricType` determines for which types of units the coverage will be measured.
+It is:
+ - `LINE`
+ - `INSTRUCTION`
+ - `BRANCH`
+
+For comparison with the specified boundaries, the number of covered (executed) or skipped (not executed) units should be aggregated into one number.
+`AggregationType` determines exactly how the current measurement value will be calculated:
+- `COVERED_COUNT` - the total number of units of code that were executed
+- `MISSED_COUNT` - the total number of units of code that were not executed
+- `COVERED_PERCENTAGE` - is the number of covered units divided by the number of all units and multiplied by 100
+- `MISSED_PERCENTAGE` - is the number of uncovered units divided by the number of all units and multiplied by 100
+
+To calculate the current value, units are grouped by various entities.
+By default, all application units of code are grouped by a single application entity, so one current value is calculated for the entire application using the aggregating function.
+
+But you can group code units by other named entities.
+The `GroupingEntityType` type is used for this:
+- `APPLICATION` - one current coverage value for the entire application will be calculated
+- `CLASS` - the current value will be calculated individually for each class. So the bounds will be checked for each class
+- `PACKAGE` - the current value will be calculated individually for all classes in each package. So the bounds will be checked for each class
+
+## Merging reports
+If it is necessary to generate a report for a specific build variant using the Kover default report tasks, it is possible to combine the contents of the Android report and the default report.
+
+This is done by configuring reports by default
```

Comment:
I don't get it, what 'by default' means here? Maybe you meant 'with mergeWith() in the defaults block'?